### ISSUE TITLE
multiline comment display error

### ISSUE THREAD

**Issue by listnukira** [hexojs/hexo#1466]
I am using latest hexo (3.1.1), default theme for test, it have same code comment problem mentioned in #774

I'm trying to modify hexo-util/lib/highlight.js, it seems code comment display correctly but I'm not sure is this a good solution

```diff
@@ -28,8 +28,8 @@
     line = lines[i];
     if (tab) line = replaceTabs(line, tab);

-    numbers += '<span class="line">' + (firstLine + i) + '</span><br>';
-    content += '<span class="line">' + line + '</span><br>';
+    numbers += (firstLine + i) + '<br>';
+    content += line + '<br>';
   }

   result += '<figure class="highlight' + (data.language ? ' ' + data.language : '') + '">';
@@ -41,10 +41,10 @@
   result += '<table><tr>';

   if (gutter){
-    result += '<td class="gutter"><pre>' + numbers + '</pre></td>';
+    result += '<td class="gutter"><pre><span class="line">' + numbers + '</span></pre></td>';
   }

-  result += '<td class="code"><pre>' + content + '</pre></td>';
+  result += '<td class="code"><pre><span class="line">' + content + '</span></pre></td>';
   result += '</tr></table></figure>';

   return result;
```

**Comment by leesei** [comment]
Can you post the code block that is causing the issue?

**Comment by tommy351** [comment]
I think this is related to https://github.com/hexojs/hexo-util/issues/4 and it has been fixed in hexo-util@0.2.0.
